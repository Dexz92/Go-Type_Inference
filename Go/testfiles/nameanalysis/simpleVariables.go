func main() {
  var a int = 1;
  var a = 2;
  var b int = 2;
  b := 3;
  c := 4;
  var c int = 1;
}
