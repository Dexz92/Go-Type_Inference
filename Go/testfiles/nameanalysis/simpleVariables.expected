Error at line 3: symbol 'a' is already declared!
Error at line 5: symbol 'b' is already declared!
Error at line 7: symbol 'c' is already declared!
